dependencies.gradle

```diff
@@ -1,8 +1,10 @@
 // Add your dependencies here
 
 dependencies {
-    compile("com.github.GTNewHorizons:waila:1.5.18:dev")
-    compileOnly("curse.maven:minefactory-reloaded-66672:2366150")
-    compileOnly("com.github.GTNewHorizons:ForestryMC:4.4.4:dev")
-    compileOnly("com.mod-buildcraft:buildcraft:7.1.23:dev") // Needed by Forestry
+    compile('com.github.GTNewHorizons:waila:1.5.19:dev')
+
+    compileOnly('com.github.GTNewHorizons:ForestryMC:4.4.6:dev')
+    compileOnly('curse.maven:minefactory-reloaded-66672:2366150')
+
+    runtime('com.github.GTNewHorizons:TinkersConstruct:1.9.0.13-GTNH:dev')
```

Comment:
Is it really needed to runtime TiCon just to test a single "fearure"?

Reply:
Whoops meant to remove that after testing

Reply:
In future just put the dev jar in your run/mods folder :P (along with all of its deps...)

Reply:
Then I have to find the mod and download it etc etc - this + uodate_deps.py finds that all for me heh